src/logger/simple.py

```diff
@@ -0,0 +1,121 @@
+import time
+from collections import defaultdict
+
+from loguru import logger
+from torch import Tensor
+from typing import Optional, Dict
+import wandb
+
+
+class Logger():
+    def __init__(self, print_freq: int = 100, image_freq: int = 1000, tb_path: str = None):
+        self.print_freq: int = print_freq
+        self.image_freq: int = image_freq
+        self.loss_buff: dict[str, dict[str, list]] = defaultdict()
+        self.loss_buff['values'] = defaultdict(list)
+        self.loss_buff['sumlast'] = defaultdict(int)
+        self.loss_buff['sum'] = defaultdict(int)
+        self.metrics_buff: dict[str, dict[str, list]] = defaultdict()
+        self.metrics_buff['values'] = defaultdict(list)
+        self.metrics_buff['sumlast'] = defaultdict(int)
+        self.metrics_buff['sum'] = defaultdict(int)
+        self.tb_path: str = tb_path
+        self.train_iter = 1
+        self.val_iter = 1
+        self.wandb = wandb.init(project="TextDeepFake", entity="sphericalpotatoinvacuum")
+
+    def log_train(self, losses: Optional[Dict[str, float]] = None, images: Optional[Dict[str, Tensor]] = None):
+        if self.train_iter == 1:
+            self.start_time = time.time()
+            logger.info(
+                'Training started'
+            )
+        if losses:
+            for loss_name, loss_value in losses.items():
+                self.loss_buff['values'][loss_name] += [loss_value]
+                self.loss_buff['sumlast'][loss_name] += loss_value
+
+        if self.train_iter % self.print_freq == 0:
```

Comment:
We log something once in `print_freq` batches but counter goes through all epochs. Maybe we need to clear it between epochs?